AWSSDKSwiftCLI/Sources/AWSCLIUtils/Version.swift

```diff
@@ -7,15 +7,46 @@
 
 import Foundation
 import ArgumentParser
-import PackageDescription
 
-// MARK: - Decodable
+public struct Version: Equatable {
+    public let major: Int
+    public let minor: Int
+    public let patch: Int
+
+    private var versionString: String { "\(major).\(minor).\(patch)" }
+
+    init(_ major: Int, _ minor: Int, _ patch: Int) {
+        self.major = major
+        self.minor = minor
+        self.patch = patch
+    }
+
```

Comment:
Note that creating a `Version` from a string is now a non-optional initializer that throws if the string is not a valid semantic version identifier.